### PR TITLE
Add a note about teams as flag maintainers - Expected pub date 2022-12-05

### PR DESCRIPTION
**Pull request by charliepark**
We're adding functionality to enable teams to be the flag's maintainers. We're hoping to launch this feature next Monday, November 28th, but recognize that Thanksgiving might create some issues there, so it could get pushed back.

We definitely want to add the copy included in this PR (obviously, open to tweaks / suggestions). Would also like to add some screenshots, but wanted to be mindful of the "avoid screenshots unless necessary" guidance. Would love your thoughts on that! (Also, as this is enterprise-only, does that confuse things for screenshots?)

### CODE REVIEW

File: src/content/topics/home/using-flags/flag-settings.mdx
Comment (mmrj) [inline src/content/topics/home/using-flags/flag-settings.mdx:49]:
```suggestion
Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that if one member leaves the team, the flag still has other maintainers.
```

I think it's fine to add a screenshot here if you'd like one @charliepark, as long as you crop it fairly closely. It's not a problem to add a screenshot that's for Enterprise, although we may want to mention that in the caption in addition to here in preceding paragraph. 

Does the `flag-settings-all-environments.png` (line 43) need to be updated based on this feature?

**Review by mmrj — CHANGES_REQUESTED** [review]

File: src/content/topics/home/using-flags/flag-settings.mdx
Comment (charliepark) [reply src/content/topics/home/using-flags/flag-settings.mdx]: The image currently on the page isn't wrong … it's just showing a single member as flag maintainer, which is still a valid state that a flag could be in (either enterprise or not). So we can leave that one as-is, I think.

File: src/content/topics/home/using-flags/flag-settings.mdx
Comment (mmrj) [inline src/content/topics/home/using-flags/flag-settings.mdx:53]:
```suggestion
You can update the flag's maintainer on the flag's **Settings** tab. You can also update the maintainer by clicking the pencil icon from the header of any of the flag's tabs:

![The flag's maintainer and "pencil" icon.](edit-flag-maintainer.png)
```

Style nits -- active voice, and our captions [describe what the reader sees](https://launchdarkly.atlassian.net/wiki/spaces/PDH/pages/342786398/Style+guide+Tech+Writing+Documentation#Screenshot-captions), rather than the action they might do from here.

**Review by mmrj — CHANGES_REQUESTED** [review]

File: src/content/topics/home/using-flags/flag-settings.mdx
Comment (charliepark) [reply src/content/topics/home/using-flags/flag-settings.mdx:53]: Ah, thank you!

**Review by mmrj — APPROVED** [review]

**Comment by charliepark** [comment]
Will merge this after we flip the flag, which might be tomorrow, might be Monday the 5th. Thank you!

**Comment by charliepark** [comment]
New pub date: 2022-12-05

**Pull request merged**
